dubbo-config/dubbo-config-spring/src/test/java/org/apache/dubbo/config/spring/beans/factory/annotation/AnnotationPropertyValuesAdapterTest.java

```diff
@@ -66,19 +65,9 @@ public void test() {
 
         DefaultConversionService conversionService = new DefaultConversionService();
 
-        conversionService.addConverter(new Converter<String[], String>() {
-            @Override
-            public String convert(String[] source) {
-                return arrayToCommaDelimitedString(source);
-            }
-        });
-
-        conversionService.addConverter(new Converter<String[], Map<String, String>>() {
-            @Override
-            public Map<String, String> convert(String[] source) {
-                return CollectionUtils.toStringMap(source);
-            }
-        });
+        conversionService.addConverter((Converter<String[], String>) StringUtils::arrayToCommaDelimitedString);
```

Comment:
@igor-suhorukov, this change leads to CI failure. It looks like lambda lost generic info (actualTypeArguments).

pls. check org.springframework.core.SerializableTypeWrapper#forGenericInterfaces

```java
	public static Type[] forGenericInterfaces(final Class<?> type) {
		Type[] result = new Type[type.getGenericInterfaces().length];
		for (int i = 0; i < result.length; i++) {
			final int index = i;
			result[i] = forTypeProvider(new SimpleTypeProvider() {
				@Override
				public Type getType() {
					return type.getGenericInterfaces()[index];
				}
			});
		}
		return result;
	}
```

Reply:
@beiwei30 thank you for assistance